extensions/sql-database-projects/src/models/deploy/deployService.ts

```diff
@@ -181,7 +181,12 @@ export class DeployService {
 		// Clean up existing docker image
 		const containerIds = await this.getCurrentDockerContainer(imageLabel);
 		if (containerIds.length > 0) {
-			const result = await vscode.window.showWarningMessage(constants.containerAlreadyExistForProject, constants.yesString, constants.noString);
+			// TODO - Fix this going away
```

Comment:
is this comment outdated? Doesn't the quickpick fix the problem of the warning message going away?